UnderWaterPhoto/Scenes/AuthScreen/AuthViewController.swift

```diff
@@ -18,35 +23,117 @@ final class AuthViewController: UIViewController {
     var presenter: AuthPresenterProtocol?
     
     // MARK: PrivateProperties
+    private var isExpandedLoginButton: Bool = false
+    private var passwordTextFieldCenterYConstraint: Constraint?
+    private var emailTextFieldCenterYConstraint: Constraint?
+    private var loginButtonCenterYConstraint: Constraint?
+    private var registrationButtonCenterYConstrain: Constraint?
+    
+    private lazy var logoLabel: UILabel = {
```

Comment:
Ко всем юай элементам, кроме кнопок, можно не использовать модификаторы lazy, достаточно private let. Плохого в этом ничего нет, но код выглядит грязнее
```suggestion
    private let logoLabel: UILabel = {
```

Reply:
выполнил